fractals/sunqua-peak/index.md

```diff
@@ -168,11 +176,15 @@ If you do not manage to bring her to 66%, she will dash twice and get <Effect na
 <GifPlayer caption="Dodge the Meteor AoE" url="https://cdn.discordapp.com/attachments/662333637143822339/759371940090871838/meteor.webm"/>
 </GridItem>
 <GridItem xs="12" sm="7">
-After bringing the boss down to 66%, she gets <Effect name="Invulnerability"/> and spawns meteor impact AoEs. Outrun them like a turtle (there is enough time even with roleplay walk). After the third set of impacts the boss reappears. Be aware, there is a fourth set of 4 meteor impacts in a square formation around the boss. Dodge them! Two players are also targeted by meteor AoEs that fall shortly after the final 4. They should be placed outside of the stack (in a non static team), or just ignore them by dodging at the right time (in a coordinated team). 
+After bringing the boss down to 66%, she gets <Effect name="Invulnerability"/> and spawns meteor impact AoEs. Outrun them like a turtle. After the third set of impacts the boss reappears. Be aware, there is a fourth set of 4 meteor impacts in a square formation around the boss. Dodge them! Two players are also targeted by meteor AoEs that fall shortly after the final 4. They should be placed outside of the stack (in a non static team), or just ignore them by dodging at the right time (in a coordinated team). 
 
-After all meteors have fallen, an AoE around the boss starts to expand, launching a powerful attack that downs/strips mistlock from anyone caught in the blast. Seek shelter behind a rock (this is the same pattern as in the Fury of the Mountain encounter). Be aware, there will only be one rock in Challenge Mode. The ''fake'' rocks are ones that glow red before crumbling. 
+</GridItem>
+
+<GridItem xs="12" sm="12">
+After all meteors have fallen, an AoE around the boss starts to expand, launching a powerful attack that downs/strips mistlock from anyone caught in the blast. Seek shelter behind a rock (this is the same pattern as in the Fury of the Mountain encounter). Be aware, there will only be one rock in Challenge Mode. The ''fake'' rocks are ones that glow red before crumbling. The safe rock is **always** the south west one on minimap.
 
```

Comment:
So always been south west was my hunch too.
But then we saw it on north (the only time after the patch this week we had low dps cuz of ressing)
So now i just tend to think now its just rng :(